examples/lib/defaultHtmlSequencerUi.js

```diff
@@ -27,7 +26,8 @@ function DefaultHtmlSequencerUi(_sequencer, options) {
   function selectNewStepUi() {
     var m = $(addStepSel + ' select').val();
     if(!m) m = arguments[0];
```

Comment:
I think `arguments[0]` is undefined or not useful anyway, so you can just remove this line and instead add
`if (m) $(addStep...`